students/eowyn/session07/html-rendering/html_render.py

```diff
@@ -1,35 +1,32 @@
-
-
 class Element():
+    # Pretty-print HTML files with increasing indent levels
+    # for each depth of tag.
     tag = 'html'
-    indent = 4 * ' '
+    extra_indent = 4 * ' '
```

Comment:
is this "extra"? or the basic amount.

If you want to be more explicit, maybe "indent_size" or something...

though if "extra' means something to you -- go for it!